movieland-web/src/main/java/com/gromoks/movieland/web/util/JsonJacksonConverter.java

```diff
@@ -20,7 +20,7 @@
         try {
             return objectMapper.writerWithView(clazz).writeValueAsString(list);
         } catch (JsonProcessingException e) {
-            log.error("Error", e);
+            log.error("Error in Json convert based on view", e);
             throw new RuntimeException(e);

```

Comment:
throw new RuntimeException("Error in Json convert based on view",e);

Reply:
Done